### DISCUSSION TITLE
Send help.

### DISCUSSION THREAD

**Discussion by NoobyIsHere** [ppy/osu#14540] [Q&A]
When i try to import a osu lazer skin it failed to import i tryed everything android 10 mobile with a keyboard and mouse send help plz

**Comment by bdach** [comment]
please describe what you have tried and what were the results (by attaching logs, or a video, or anything). we cannot help you otherwise.

**Comment by jhey29** [comment]
Importing beatmaps (I didn't try for skins yet) only works when osu! doesn't have to boot while importing, on my Android 10 phone.
This is 1 issues: 
0. When I upload an osz file I modified on my laptop to my phone, then use either Share to osu! or Open with osu! on that file, the file doesn't import. I am only alerted that "an unobserved error has occurred". The last few log files in the zip are just me using share, then closing the game after being in settings for a bit.
[logs_today_manual.zip](https://github.com/user-attachments/files/16660550/logs_today_manual.zip)
Wait OOPS it does work. But only with Open with, and if osu is already running, because Share appears to start a new instance of osu inside my file manager somehow.
![Screenshot_20240819-15](https://github.com/user-attachments/assets/5adf6bc2-7e18-426c-a544-ae3ce7a60022)
*Image of two osu! tabs, one being labeled with the files icon, the other is almost completely offscreen*



1. I don't know whether Debug -> Import files is even supposed to be used for this, but it and the other beatmap image / audio file dialogs do not work. They display some android- or osu-internal directories the app probably has opened for other reasons like "dev" and "com.android.media", and even "storage", but I can't get to any directory I can put files in. If I interpret this correctly, because the Internal shared storage is at /storage/emulated/0/, but android is really stingy with permissions outside of this area, I can click on "emulated" inside "storage", but because I am not allowed to view it, I can't click on "0" anywhere. I recursively clicked all the avaliable folders and couldn't find the Internal shared storage anywhere else either. I obviously can't enter the path into a textbox because there is no textbox.

**Reply by jhey29** [reply]
recently Files by Google somehow stopped knowing that osu opens .osz-files (I didn't test anything else). Other File Managers that don't care about this can still do Open With correctly though.

**Reply by shiumano** [reply]
In the context menu in the list of downloaded files in Chrome, you will find “Share”. If the file was downloaded in the browser, you can download it from there. (You must open osu once to address #28790.)
Also, `/storage/emulated/0/Android/media/sh.ppy.osulazer/` is accessible from both file manager and osu. It is not the right way, but for now this hack can also be used.
